### PR TITLE
add php5.3

### PR DESCRIPTION
**Pull request by nguyen-hieu-zalora**

### CODE REVIEW

File: sdk.nix [hunk 1]
```diff
@@ -40,7 +40,7 @@ rec {
     packageOverrides = pkgs: rec {
       inherit (ugpkgs) docker go imagemagick linux nix;
       mysql = ugpkgs.mariadb;
-      php = ugpkgs.php54;
+      php = ugpkgs.php53;
```
Comment (proger) [inline sdk.nix:43]: i think that's redundant as we're going to override php only for one venture seafas

Comment (proger) [inline sdk.nix:43]: better add `php53` verbatim to the sdk: https://github.com/zalora/microgram/blob/master/sdk.nix#L138

**Comment by phunehehe** [comment]
with https://github.com/zalora/microgram/pull/43 sdk.nix is the only change that is needed here

**Comment by phunehehe** [comment]
(obsoleted by https://github.com/zalora/microgram/pull/43 and https://github.com/phunehehe/microgram/commit/6869584f405e3f0b9a252a77b8d5d576be57241e)

**Pull request closed without merging**